package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "lcdgame",
+  "version": "0.3.4",
+  "description": "LCD game library in html5/javascript, to recreate simulations of 80s Game & Watch by Nintendo and other electronic handheld games like Tandy, Sunwing, Mini Arcade etc.",
+  "main": "bin/lcdgame.js",
+  "bin": {
+    "lcdgame": "lcdgame - kopie (2).js"
+  },
+  "scripts": {
+    "minify": "uglifyjs bin/lcdgame.js --comments /^!/ --source-map --mangle --output bin/lcdgame.min.js",
```

Comment:
Added a minification script as well. I suppose you have your own local script to do this, thought it would be easy.

Eventually a bundler would be nice too, to generate lcdgame.js and the sourcemaps from the separate modules too - for local development.